give/src/components/Tabs/Tabs.js

```diff
@@ -7,49 +12,55 @@ export const Tabs = ({
 }) => {
   if (content.length === 0) return null;
 
-  const classes = className !== "" ? " " + className : className;
-  const dotClass = dot ? " cru-tabs-dot" : "";
-
   return (
-    <div className={"tabs cru-tabs" + dotClass + classes}>
-      <div className="cmp-tabs">
-        <ol className="cmp-tabs__tablist">
-          {content.map((tab, index) => {
-            let active = index === 0 ? " cmp-tabs__tab--active" : "";
-            return (
-              <li className={"cmp-tabs__tab" + active} key={tab.tab}>
-                {tab.tab}
-              </li>
-            );
-          })}
-        </ol>
-        {content.map((tabpanel, index) => {
-          let active = index === 0 ? " cmp-tabs__tabpanel--active" : "";
+    <ComponentWrapper
+      type="tabs"
+      className={`${dot ? "cru-tabs-dot " : ""}${className}`}
+    >
+      <ol className="cmp-tabs__tablist">
+        {content.map((tab, index) => {
```

Comment:
In the end, the content will be provided through AEM, so this will undoubtedly have to change. For now, I opted for the current solution so that in my `content` variable, I could neatly keep tab labels and tab content grouped together.